Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+
+[requires]
+python_version = "3.8"
```

Comment:
Does this line mean 3.8 is required now to generate the docs?

The min version of Python we currently support is 3.6, which means _I think_ we want to continue to build our packages using 3.6 (according to https://docs.travis-ci.com/user/languages/python/#specifying-python-versions, 3.6 is the default version of Python on Travis CI).

If we're now requiring 3.8 to generate the docs, won't we run into issues when releasing a new version of a package, since it'll be using Python 3.6 to regen the docs?

Reply:
Yeah this whole file was an unintentional addition. I have removed it.